src/utils.js

```diff
@@ -118,6 +118,29 @@ const ensureLabelExists = async (octokit, repo, owner, name, color) => {
   }
 };
 
+const fetchTeamMembers = async (octokit, organization, teams) => {
+  const result = [];
+
+  for (const slug of teams) {
+    const memberIterator = octokit.paginate.iterator(
+      "GET /orgs/{org}/teams/{team_slug}/members",
+      {
+        org: organization,
+        team_slug: slug,
+        per_page: 100,
+      }
+    )
+
+    for await (const { data: members } of memberIterator)   {
+      for (const member of members) {
+        result.push(member.login)
+      }
+    }
+  }
```

Comment:
I could be wrong 🤔, but I believe this `await` will hold up iteration of `teams` as well as pages. Is that the desired behavior? That could be appropriate for rate limiting, but unless we strongly suspect it's an issue we can decrease runtime (significantly for multiple paginated teams) by fetching and processing teams concurrently, along the lines of this quasi-pseudo-code 😄:

```javascript
const fetchTeams = async (octokit, organization, teams) => {
  const results = await Promise.allSettled(
    teams.map(
      slug => fetchTeamMembers(octokit, organization, slug)
    )
  )
  return [...results]
}
```

Reply:
Good idea! Let me test this to make sure that it works; if it does I will commit it. I copied the previous code from an Octokit example, but I agree that we should avoid the unnecessary serialized behaviour if possible.

Reply:
Fixed in 9335288d2abd6b1560d8240fff31eecb11af9a47.